src/backends/perf_pt/mod.rs

```diff
@@ -403,6 +403,15 @@ impl Default for PerfPTThreadTracer {
     }
 }
 
+impl Drop for PerfPTThreadTracer {
+    fn drop(&mut self) {
+        if matches!(self.state, TracerState::Started) {
```

Comment:
I guess that can't be `==` or you'd have used it :P

Reply:
I haven't tried, but last time I used `==` on an enum it didn't work.

Reply:
Hm, seems to work. So `matches!` is probably only needed for more complex matches.

Reply:
Ah, that was it!